packages/nim/build.sh

```diff
@@ -0,0 +1,56 @@
+TERMUX_PKG_HOMEPAGE=https://nim-lang.org/
+TERMUX_PKG_DESCRIPTION="Nim programming language compiler"
+TERMUX_PKG_VERSION=0.18.0
+TERMUX_PKG_SRCURL=https://nim-lang.org/download/nim-$TERMUX_PKG_VERSION.tar.xz
+TERMUX_PKG_SHA256=45c74adb35f08dfa9add1112ae17330e5d902ebb4a36e7046caee8b79e6f3bd0
+TERMUX_PKG_HOSTBUILD=yes
+TERMUX_PKG_BUILD_IN_SRC=yes
+TERMUX_PKG_DEPENDS="git, clang, libandroid-glob"
+termux_step_host_build() {
+	cp ../src/* -r ./
+	make -j $TERMUX_MAKE_PROCESSES CC=gcc LD=gcc
+	make install
```

Comment:
For the third time, correct me if I'm wrong, but won't this install nim to the host system?  It seems this step is completely unnecessary since you're adding this local build to your path later on in the build.